frontend/src/components/StatsPage.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Container, Button, Table } from "reactstrap";
+import { PlayerStats } from "../models/stats.model";
+import { RootSchema } from "../store/root-schema";
+import { Dispatch } from "../store";
+import { changeView } from "../store/actions";
+import { connect } from "react-redux";
+
+interface Props {
+  username: string;
+  stats: PlayerStats;
+  dispatch: Dispatch;
+}
+
+const StatsPage: React.FunctionComponent<Props> = ({
+  username,
+  stats,
+  dispatch,
+}: Props) => {
+  const back = () => {
+    dispatch(changeView("login"));
+  };
+  const gameSuccessRatio =
+    stats && stats.playedGames ? (stats.wonGames ?? 0) / stats.playedGames : "No info";
+  const roundsSuccessRatio =
+    stats && stats.playedRounds
+      ? (stats.wonRounds ?? 0) / stats.playedRounds
+      : "No info";
+  return (
+    <Container>
+      <h1 className="stats-header">Stats for user {username}</h1>
```

Comment:
Instead of `className="stats-header"` you could use `Caption` component. But it's up to you.